SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+| Version            | Supported          |
+| ------------------ | ------------------ |
+| 1.x.x (iOS/macOS)  | :white_check_mark: |
+| 0.x.x              | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+As we don't think there are any possible critical vulnerabilities, use Github Issues to report them.
```

Comment:
... If there somehow still is a important security vulnerability, please contact GameTec-live or Foxushka immideatly. This can be done via for example discord.